pkg/keyring/keyring_windows.go

```diff
@@ -24,21 +24,15 @@ import (
 // ClearCredentials removes any existing items in the keychain,
 // it will ignore if not found errors
 func ClearCredentials(prefix string) error {
-	for _, k := range []string{username, password} {
+	for _, k := range []string{username, password, bearer} {
 		if err := deleteSecret(format(prefix, k)); err != nil {
 			if !errors.Is(err, zkeyring.ErrNotFound) {
 				return err
 			}
 		}
 	}
-	p, err := filepath.Abs(fmt.Sprintf("%s/%s", profiles.GetStorageDirectory(), format(prefix, bearer)))
-	if err != nil {
-		return err
-	}
-	if ok, err := util.FileExists(p); err == nil && ok {
-		if err := os.Remove(p); err != nil {
-			return err
-		}
+	if _, ok := os.LookupEnv("SDPCTL_BEARER"); ok {
```

Comment:
i was expecting you to call the `DeleteBearer() `method from inside `ClearCredentials()`, how come it was removed?

Reply:
Sorry, can't remember really what my thought was, but I added it now.